### PR TITLE
Refactor how-to section

### PR DESCRIPTION
**Pull request by fabienrohrer**
Result:  https://cyberbotics.com/doc/guide/how-to?version=update-howto-samples-section

### CODE REVIEW

**Comment by fabienrohrer** [comment]
English fixed by Tom.

**Comment by fabienrohrer** [comment]
👍

File: guide/how-to.md [hunk 1]
```diff
@@ -1,87 +1,153 @@
...
+Note that more completed Ackermann steering geometry can be achieved using the `AckermannVehicle.proto` model, and the `vehicle` library (cf. the [automobile book](https://www.cyberbotics.com/doc/automobile/index)).
+
+
+### gui\_tracker.wbt
+
+**Keywords**: GUI, custom Qt-based robot window
+
+![gui_tracker.png](images/gui_tracker.png) This example shows the use of the deprecated Qt-based system to create custom windows.
+A Supervisor is getting the position of five e-pucks moving randomly, and this information is sent to the custom robot window plugin called `tracking window`.
+This window is based on Qt (embedded in Webots) and the `Qt utils` helper library.
```
Comment (omichel) [inline guide/how-to.md]: We should update this example to use the HTML robot window instead...

Comment (fabienrohrer) [reply guide/how-to.md]: I agree; I created a hand-written list of samples which should be completed or added, this one is probably the most important.

**Pull request merged**